pyproject.toml

```diff
@@ -1,3 +1,10 @@
+[build-system]
+requires = ["setuptools>=48", "setuptools_scm[toml]>=6.3.1"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "src/dvclive/_version.py"
```

Comment:
Does dvclive need to provide version info?

```pycon
>>> from importlib.metadata import version; version('dvclive')
```

Users can get this with above ^.

Reply:
Not really required tbh, could get rid of it if it's not adding any value

Reply:
Got rid of it, as well as getting rid of `src/dvclive/version.py`